[Latest git version] Compilation error

The latest git version demands `-Wno-error` to compile:
```
/media-gfx/darktable-9999/work/darktable-9999/src/views/knight.c
/var/tmp/portage/media-gfx/darktable-9999/work/darktable-9999/src/views/knight.c: In function «_event_loop»:
/var/tmp/portage/media-gfx/darktable-9999/work/darktable-9999/src/views/knight.c:951:10: error: «res», probably is used uninitialized [-Werror=maybe-uninitialized]
  951 |   return res;
      |          ^~~

```

--- a/src/views/knight.c
+++ b/src/views/knight.c
@@ -934,7 +934,7 @@ static gboolean _event_loop_animation(dt_knight_t *d)
 static gboolean _event_loop(gpointer user_data)
 {
   dt_knight_t *d = (dt_knight_t *)user_data;
-  gboolean res;
+  gboolean res = FALSE;  // silence warning about unitialized res
   switch(d->game_state)
   {
     case INTRO:

